nordic-epub3-dtbook-migrator/src/test/xprocspec/dtbook-to-html.step.xprocspec

```diff
@@ -25,10 +29,10 @@
             <x:document type="port" port="in-memory.out"/>
         </x:context>
         <x:expect type="count" label="There should be exactly one output document" min="1" max="1"/>
-        <x:expect type="xpath" label="The output document (the intermediary HTML) should have an xml:base attribute with filename C00000.xhtml" test="tokenize(/*/@xml:base,'/')[last()]"
+        <x:expect type="xpath" label="The output document (the intermediary HTML) should have a base URI with filename C00000.xhtml" test="tokenize(/*/base-uri(.),'/')[last()]"
             equals="'C00000.xhtml'"/>
-        <x:expect type="xpath" label="The output document (the intermediary HTML) should have an xml:base attribute pointing to the temp-dir directory" test="/*/@xml:base"
-            equals="concat($temp-dir,'temp-dir/C00000.xhtml')"/>
+        <x:expect type="xpath" label="The output document (the intermediary HTML) should have a base URI pointing to the temp-dir directory" test="/*/base-uri(.)"
+            equals="concat($temp-dir,'temp-dir/C00000.xhtml')" pending="the base URI is normalized"/>
```

Comment:
Does this test have to be pending? Do we need to change something to make this work?

Reply:
Oh, right, forgot about this change. I just checked it again the pending doesn't seem to be needed. Can be changed in https://github.com/nlbdev/nordic-epub3-dtbook-migrator/pull/385.

Reply:
Ok, great :+1: